cj3.txt

```diff
@@ -97454,6 +97454,7 @@ nkhqm  𭚕
 nkhv   𭚓
 nkji   𭐳
 nkks   𭄶
+nfleg  𮫹
 nkleg  𮫹

```

Comment:
`nkleg` is good because the J-Source glyph has 𩵋,

![image](https://user-images.githubusercontent.com/3607926/110243255-97abc180-7f27-11eb-846f-235c5bed634f.png)

However in [IRGN 2425](https://appsrv.cse.cuhk.edu.hk/~irg/irg/irg53/IRGN2425ucv-summary.pdf) it is well established that 𩵋/魚 is unifiable pairs, which means ⿰魚坚 will never be encoded separately and should always be unified with 𮫹. Given that BabelHan Stone has ⿰魚坚, which means there are such evidences, I think we should leave `nkleg` as-is and add `nfleg` for ⿰魚坚.